config/rbac/manager_extraroles_patch.yaml

```diff
@@ -2,11 +2,11 @@
   path: /rules/-
   value:
     apiGroups:
-      - cluster.x-k8s.io
+      - etcdcluster.cluster.x-k8s.io
```

Comment:
are `etcdcluster.cluster.x-k8s.io` and `controlplane.cluster.x-k8s.io`  still needed when u have `cluster.x-k8s.io` defined?

Reply:
yes